mavros_msgs/msg/OnboardComputerStatus.msg

```diff
@@ -0,0 +1,26 @@
+# Mavros message: ONBOARDCOMPUTERSTATUS
+
+std_msgs/Header header
+
+uint8 component               # See enum MAV_COMPONENT
+
+uint64 time_usec            # [us] Timestamp (UNIX Epoch time or time since system boot)
```

Comment:
Probably `header.stamp` should be used for mavlink message.

Reply:
right, that makes sense